File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cs
using System;

public class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        // 1.
        string a1 = a.ToString();
        string b1 = b.ToString();
        
        // 2.
        int answer1 = int.Parse(a1+b1);
        int answer2 = int.Parse(b1+a1);
        
        // 3.
        if(answer1 >= answer2) {
            answer = answer1;
        }
        else {
            answer = answer2;
        }
        return answer;
    }
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) 
{
    string a_str = to_string(a);
    string b_str = to_string(b);
    
    string sumAB = a_str + b_str;
    string sumBA = b_str + a_str;
    string result = "";
    if(stoi(sumAB) > stoi(sumBA))
    {
        result = sumAB;
    }
    else if(stoi(sumAB) < stoi(sumBA))
    {
        result = sumBA;
    }
    else
    {
        result = sumAB;
    }
    return stoi(result);
}